nexus/tests/integration_tests/instances.rs

```diff
@@ -4325,6 +4334,7 @@ async fn test_boot_disk_can_be_changed(cptestctx: &ControlPlaneTestContext) {
         RequestBuilder::new(client, http::Method::PUT, &url_instance_update)
             .body(Some(&params::InstanceUpdate {
                 boot_disk: Some(disks[1].identity.id.into()),
+                auto_restart_policy: None,
```

Comment:
one of these really ought to check that `auto_restart_policy` on the returned instance is _actually_ `None` after the update. this one's in `test_boot_disk_can_be_changed` which maybe should just turn into `test_instance_can_be_updated`?